pmrl.py

```diff
@@ -142,97 +146,102 @@ def main() -> None:
         for i in range(num_mobs)
     ]
     mobs_hp = [random.randint(1, 5) for i in range(num_mobs)]
-    # pick a random tile to place the player
-    tile = '#'
     player_coords = choose_random_open_tile(map_tiles, occupied_coords)
     player_hp = 10
     libtcod.console_set_custom_font('arial10x10.png', libtcod.FONT_TYPE_GRAYSCALE | libtcod.FONT_LAYOUT_TCOD)
-    libtcod.console_init_root(screen_width, screen_height, 'pmrl', False)
```

Comment:
This all looks like a bunch of changes, but it's not. 

It's mostly persisting the root console returned by `libtcod.console_init_root()` and using it as a python context manager (indenting all of the remaining stuff in one tab)